### ISSUE TITLE
Regression with supported JSON schema on broker catalog 

### ISSUE THREAD

**Issue by renelehmann** [cloudfoundry/capi-release#222]
## Issue

A regression has been detected between using cf-deployment v17.0.0 with capi-release **1.120.0** and cf-deployment v17.1.0 with capi-release **1.123.0**.  
Newer JSON schemas than draft-04 on the broker catalog which have been supported before aren't supported anymore.

On updating the broker CC errors e.g.:
Schema service_binding.create.parameters is not valid. Custom meta schemas are not supported.

With a catalog on the broker like that:
```
                  "service_binding": {
                    "create": {
                      "parameters": {
                        "$schema": "http://json-schema.org/draft-06/schema#",

```

## Context
cf-deployment v17.1.0 with capi-release 1.123.0
broker with a catalog and parameters referring $schema draft-06.

Updating the catalog on CC errors (cf-cli 7.3.0):
```
cf update-service-broker broker brokeruser password https://broker.example.com
Updating service broker broker as ...
Job failed: Service broker catalog is invalid:
Service broker
  Plan free
    Schemas
      Schema service_binding.create.parameters is not valid. Custom meta schemas are not supported.

FAILED

```

Testings:
The update of CC (newer capi 1.123.0) with a catalog referring to schema draft-06 doesn't work.
The update of CC (newer capi 1.123.0) with a catalog referring to an older JSON schema draft-04 works.
The update of CC (older capi 1.120.0) with a catalog referring to schema draft-06 works.  

## Expected result
capi should support at least JSON $schema draft-06 as it was before the regression.

## Current result
capi-releases with and between 1.121.0 and 1.123.0 doesn't support https://json-schema.org/  draft-06 anymore but does support JSON schema draft-04.

**Comment by philippthun** [comment]
Hi @renelehmann, it seems you have to adapt the JSON itself. Cloud Controller now uses `json-schema` gem version 2.8.1; according to the [changelog](https://github.com/voxpupuli/json-schema/blob/master/CHANGELOG.md), `draft-06/schema#` is not valid anymore:

> Corrected the draft6 schema id to http://json-schema.org/draft/schema#

**Comment by renelehmann** [comment]
Thanks a lot @philippthun for the hint. It looks like we have a breaking change if "draft-06" is used and "draft" cannot be referred on the previous capi versions.

**Comment by philippthun** [comment]
cloudfoundry/cloud_controller_ng/issues/2677

**Comment by philippthun** [comment]
The `json-schema` gem has been [reverted](https://github.com/cloudfoundry/cloud_controller_ng/pull/2710) to version 2.8.0. The link for **JSON schema draft 6** is https://json-schema.org/draft-06/schema.

This change will be included in the next capi-release (CAPI 1.128.0).

Note: As a workaround, changing the JSON schema to "draft-4" should mitigate the problem (this is also the version guaranteed to work: "Platforms MUST support at least JSON Schema draft v4.").
